src/anyvar/utils/liftover_utils.py

```diff
@@ -0,0 +1,292 @@
+    match_group = re.search(r":(?:chr)?(\d+|[XY])$", reference_alias)
+    chromosome_number: str | None = match_group.group(1) if match_group else None
+    if chromosome_number is None:
+        return None
+
+    return f"chr{chromosome_number}"
+
+
+def get_from_and_to_assemblies(aliases: dict) -> tuple[str, str]:
+    """Determine which assembly we're starting from, and which we're lifting over to.
+
+    Takes in a dictionary containing two lists of aliases, one for each supported reference assembly (GRCh37 and GRCh38):
+    - If an assembly contains aliases for the variant, the variant is part of that assembly.
+    - If *both* assemblies contain aliases for the variant, then it is identical across assemblies and requires no liftover.
+    - If *neither* assembly contains the variant, something has gone wrong: raises an exception
+
+    :param aliases: A dictionary containing a list of aliases for both supported reference assemblies (GRCh37 and GRCh38)
+    :return: A tuple containing the assembly we're converting from and the one we're converting to
+    :raise: A ReferenceAssemblyResolutionError if neither assembly contains the variant
+    """
+    grch37, grch38 = sorted(aliases.keys())
+
+    if aliases[grch37] and not aliases[grch38]:
+        from_assembly = grch37
+        to_assembly = grch38
+    elif aliases[grch38] and not aliases[grch37]:
+        from_assembly = grch38
+        to_assembly = grch37
+    elif not aliases[grch37] and not aliases[grch38]:
+        raise UnsupportedReferenceAssemblyError
+    else:
+        raise AmbiguousReferenceAssemblyError
+
+    return from_assembly, to_assembly
+
+
+def _convert_coordinate(converter: Converter, chromosome: str, coordinate: int) -> int:
+    """Convert an individual coordinate to another reference genome. If the conversion is unsuccessful, raises a `CoordinateConversionError`
+
+    :param converter: An AGCT Converter instance.
+    :param chromosome: The chromosome number where the coordinate is found. Must be a string consisting of a) the prefix "chr", and b) a number OR "X" or "Y" -> i.e. "chr10".
+    :param coordinate: A single start or end coordinate. MUST be an `int`.
+
+    :return: A converted coordinate value as an `int`
+    :raises: A `CoordinateConversionError` if the conversion is unsuccessful.
+    """
+    converted_position = converter.convert_coordinate(
+        chromosome, coordinate, Strand.POSITIVE
+    )
+
+    # TODO: Handle cases where coordinate conversion returns negative-stranded coordinates. See Issue #197.
+    if converted_position and converted_position[0][2] == Strand.POSITIVE:
+        # TODO: Don't just return coordinates from the first result set - handle cases where coordinate map to multiple positions. See Issue #198.
+        return converted_position[0][1]
+    raise CoordinateConversionError
+
+
+def convert_position(
+    converter: Converter, chromosome: str, position: list | int
+) -> list | int:
```

Comment:
Up to you if you want to get this into the weeds of type annotations but I believe you can rewrite this to indicate that the return type is always equivalent to the `position` arg type

```python
from typing import TypeVar

_TPos = TypeVar("TPos", int, list[int])  # these are lists of ints right?

def convert_position(
    converter: Converter,
    chromosome: str,
    position: _TPos,
) -> _TPos:
```

that's pretty ugly though